### PR TITLE
Add squashfs-tools

### PR DESCRIPTION
**Pull request by lalten**

### CODE REVIEW

File: modules/squashfs-tools/metadata.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "homepage": "https://github.com/plougher/squashfs-tools",
+  "maintainers": [
+    {
+      "email": "bcr-maintainers@bazel.build",
```
Comment (Wyverald) [inline modules/squashfs-tools/metadata.json:5]: mind adding yourself here? :)

Comment (lalten) [reply modules/squashfs-tools/metadata.json]: done

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (squashfs-tools) have been updated in this PR. Please review the changes.

**Comment by meteorcloudy** [comment]
Please address the test failures.

**Review by Wyverald — APPROVED** [review]

**Pull request merged**